lib/isList/index.test.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const name = __filename.split('/').pop().split('.').shift(),
+  fn = require('./' + name),
+  expect = require('chai').expect;
+
+describe('isList', () => {
+  it('returns true if list reference', () => {
+    expect(fn('domain.com/lists/foo')).to.equal(true);
+  });
+
+  it('throws an error if the URI passed in is not a string', () => {
+    const nonStringArgument = function () {
+      return fn([0, 1, 2, 3]);
+    };
+
+    expect(nonStringArgument).to.throw(Error);
+  });
+
+  it('returns false if non-list reference', () => {
+    expect(fn('domain.com/users/foo')).to.equal(false);
+    expect(fn('domain.com/pages/foo')).to.equal(false);
+    expect(fn('domain.com/components/foo')).to.equal(false);
+    expect(fn('domain.com/components/foo/instances/bar')).to.equal(false);
+  });
+});
```

Comment:
do we also want to test that fn('domain.com/lists/') is false?

Reply:
Good point. That would be smarter but inconsistent with the behavior of the other fncs, so we should probably cut a separate PR that fixes it across all. Will create an issue.

Reply:
sounds good